compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs

```diff
@@ -46,7 +46,7 @@ version1_13 = V1 $ PointStable 13
 
 -- | The DAML-LF version used by default.
 versionDefault :: Version
-versionDefault = version1_12
+versionDefault = version1_13
```

Comment:
Aren’t you doing 1 step too many here for 1.14? We had 1.13 in preview in SDK 1.13. In SDK 1.14 we now need to make it stable and then we switch the default in SDK 1.15. Or am I missing something here?